server-src/src/main/java/com/itt/kmt/services/UserService.java

```diff
@@ -0,0 +1,110 @@
+package com.itt.kmt.services;
+
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import com.itt.kmt.models.User;
+import com.itt.kmt.repositories.UserRepository;
+
+/**
+ * Service class that acts as an intermediary between controller and the
+ * database for all basic CRUD operations. The business logic should reside in
+ * service class.
+ * @author Rakshit Rajeev
+ */
+@Service
+public class UserService {
+    /**
+     * Instance of the basic Repository implementation.
+     */
+    @Autowired
+    private UserRepository repository;
+    /**
+     * Gets the User given the email.
+     * 
+     * @param email Email of the User.
+     * @return User object matching the email.
+     */
+    public User getUserByEmail(final String email) {
+        return repository.findByEmail(email);
```

Comment:
I do NOT see any end point which makes this service call

Reply:
This is being used in User save method.